src/main/java/pl/orki/hackathon/webapp/band/entity/Band.java

```diff
@@ -37,7 +36,7 @@
     @Column(name = "image_url")
     private String imageUrl;
 
-    @ManyToMany(cascade = CascadeType.ALL)
+    @ManyToMany(cascade = CascadeType.MERGE)
```

Comment:
Yeah, `CascadeType.ALL` wasn't a good idea, but do you think `MERGE` here is enough?

Reply:
hibernate was throwing some errors so i changed it

Reply:
When these errors happened? I can create user without problems. I would remain every CascadeType, but without `CascadeType.REMOVE`